lte/gateway/c/session_manager/test/SessionStateTester.h

```diff
@@ -37,7 +37,7 @@ class SessionStateTest : public ::testing::Test {
     rule_store    = std::make_shared<StaticRuleStore>();
     session_state = std::make_shared<SessionState>(
         IMSI1, SESSION_ID_1, cfg, *rule_store, tgpp_ctx, pdp_start_time,
-        CreateSessionResponse{});
+        CreateSessionResponse{}, 0);
```

Comment:
why initialized to 0?  What is the point to initialize if we don't even have a value yet? You can just default it in the constructuor

Reply:
ShardID allocation should happen outside of SessionState I think. For testing, let's use a non-0 value though.

Reply:
So the reason was we wanted to add the ue before intitializing the session. If you look at local_enforcer->init_session I add the UE then set the shard id for the session. I may change this approach a bit, based on the template Oriol provided.